packages/blade/src/components/Input/OTPInput/OTPInput.tsx

```diff
@@ -341,9 +375,11 @@ const OTPInput = ({
         alignItems={isLabelLeftPositioned ? 'center' : undefined}
         position="relative"
       >
-        <FormLabel as="label" position={labelPosition} htmlFor={inputId}>
-          {label}
-        </FormLabel>
+        {Boolean(label) && (
+          <FormLabel as="label" position={labelPosition} htmlFor={inputId}>
+            {label as string}
```

Comment:
why is `as string` needed?

Reply:
<img width="815" alt="Screenshot 2023-07-27 at 4 54 10 PM" src="https://github.com/razorpay/blade/assets/46647141/a0f03873-3d7b-440d-8cfe-bb729b02fa37">

Reply:
Can we make the FormLabel accept string or undefined then?

Reply:
@saurabhdaware done in https://github.com/razorpay/blade/pull/1472/commits/89956db0570493fed65157842048ed6566fb0ad4